docs/modules/ROOT/pages/tmail-backend/jmap-extensions/deletedMessagesVault.adoc

```diff
@@ -8,15 +8,16 @@ mailboxes.
 
 == Additions to the capability object
 
-Servers supporting the deleted message vaul extension need to advertise it through the session capabilities:
+Servers supporting the deleted message vault extension need to advertise it through the session capabilities:
 ....
 com:linagora:params:messages:vault
 ....
 
 The associated object have the following fields:
 
- - `maxRestoredEmails`: *Number | null*, defaulting to 5. Maximum number of emails a given `EmailRecoveryAction` would
+- `maxEmailRecoveryPerRequest`: *Number | null*, defaulting to 5. Maximum number of emails a given `EmailRecoveryAction` would
 allow to recover.
+- `restorationHorizon`: *String | null*, defaulting to 15 days. Horizon past which users can no longer recover emails on their own.
```

Comment:
Please precise the format of the duration here and choose something simple!

Reply:
[index.adoc](https://github.com/linagora/tmail-backend/blob/master/docs/modules/ROOT/pages/tmail-backend/configure/index.adoc) already specifies:

```
Default to 15 days. Supported units are
`ms` (or `msec`, `msecs`),
`s` (or `sec`, `secs`, `second`, `seconds`),
`m` (or `min`, `mins`, `minute`, `minutes`),
`h` (or `hour`, `hours`),
`d` (or `day`, `days`),
`w` (or `week`, `weeks`),
`month` (or `months`),
`y` (or `year`, `years`).

Example: `emailRecoveryAction.restorationHorizon=10d`
```

is that okay ? should i put the same here ?

Reply:
Maybe formatDurationWords CF https://commons.apache.org/proper/commons-lang/javadocs/api-3.0/org/apache/commons/lang3/time/DurationFormatUtils.html

Reply:
it works amazing :)